FetchRequests/Sources/SwiftUI/FetchableRequest.swift

```diff
@@ -56,7 +56,7 @@ public struct FetchableRequest<FetchedObject: FetchableObject>: DynamicProperty
 
     private let animation: Animation?
 
-    internal var hasFetchedObjects: Bool {
+    public var hasFetchedObjects: Bool {
```

Comment:
Oh no... I just realized the backing property never got javadoc... should probably add that to `FetchedResultsControllerProtocol.hasFetchedObjects` and this one too... maybe something like:
`/// Has performFetch() completed?`

Reply:
Reason being that this property has always been a little non-obvious... it has no bearing on whether the controller is empty or not, just whether it has successfully called and finished a `performFetch()` call (and not been reset)